site/src/pages/release-notes/1.18.0.mdx

```diff
@@ -0,0 +1,129 @@
+---
+date: 2022-08-05
+---
+
+## 🌟 New features
+
+- <type://DocService> can now render Markdown and [Mermaid](https://mermaid-js.github.io/mermaid/)
+  formatted documentation for annotated services #4299 #4304
+
+```java Markdown
+@Description(value = "### Markdown format", markup = Markup.MARKDOWN)
+@Get("/markdown")
+public HttpResponse markdown() {
+  return HttpResponse.of(200);
+}
+```
+
+```java
+@Description(value = "graph TD;\n" +
+                     " A-->B;\n" +
+                     " A-->C;\n" +
+                     " B-->D;\n" +
+                     " C-->D;",
+             markup = Markup.MERMAID)
+@Get("/mermaid")
+public HttpResponse mermaid() {
+  return HttpResponse.of(200);
+}
+```
+
+- It is now possible to bind decorators to gRPC services using <type://GrpcServiceBuilder>. #4311 #4316
+
+```java
+GrpcService.builder()
+  .addService(myGrpcService, List.of(decorator)) // 👈👈👈
+  .build();
+```
+
+- gRPC users can now use an <type://UnframedGrpcErrorHandler> which provides a
+  [richer error model](https://cloud.google.com/apis/design/errors#error_model) for unframed requests.
+
+```java
+GrpcService
+  .builder()
+  .addService(myGrpcServiceImpl)
+  .unframedGrpcErrorHandler(
+    UnframedGrpcErrorHandler.ofRichJson()) // 👈👈👈
+  .build();
+```
+
+```json
+{
+  "error": {
+    "code": 2,
+    "message": "A custom message!"
+  }
+}
```

Comment:
By the way, did we decide to remove the enclosing `error` block?
```suggestion
{
  "code": 2,
  "message": "A rich error message!",
  "details": [
    {
      "@type": "type.googleapis.com/google.rpc.ErrorInfo",
      "reason": "Unexpected exception...",
      ...
    }
  ]
}
```

Reply:
Oh, I thought we didn't remove it for some reason 😅   Thanks for pointing this out 👍

Reply:
I missed the discussion. I prefer to support the open-source/community version first.
 We may add Google API compatible version later if there is a demand for it.